src/components/page-header/page-header.component.html

```diff
@@ -3,7 +3,7 @@
     <div *ngIf="!condensed" class="page-header-content">
 
         <!-- Logo/product acronym -->
-        <div class="page-header-logo-container" role="presentation" [style.backgroundColor]="logoBackground" [style.color]="logoForeground">
+        <div uxFocusIndicator class="page-header-logo-container" role="presentation" [style.backgroundColor]="logoBackground" [style.color]="logoForeground" (click)="logoClick.emit($event)">
```

Comment:
I'm wondering if it would be good to make this tabbable as a user way want to perform the "click" using the keyboard, as you can see there is no way of doing this currently:

https://plnkr.co/edit/SCvXpWj52MuKueHXGjeR?p=preview

Reply:
tab index added